startCountdown.js

```diff
@@ -0,0 +1,27 @@
+//variable which will allow us to insert out timer into right div
+var startTimer = $("#timer")
+var ableToMove = 0
+//this element connects to the start button
+$("#startButton").click( function() {
+    createTimer()
+})
+
+//function that allows us to countdown number in next function
+function countDown(number) {
+    return number - 1
+}
+
+//the function for the timer
```

Comment:
edit to be more specific. i.e. the function to start the count in timer to commence the game